src/imex.rs

```diff
@@ -460,6 +460,8 @@ async fn import_backup(
         context.get_dbfile().display()
     );
 
+    context.sql.config_cache.write().await.clear();
```

Comment:
does this implicitly get dropped because its not stored in a variable?

Reply:
Yes, it locks the cache, clears the cache and unlocks.